Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -204,6 +207,17 @@ export default function ModelExperimental(options) {
   this._attenuation = pointCloudShading.attenuation;
   this._pointCloudShading = pointCloudShading;
 
+  // If the given clipping planes don't have an owner, make this model its owner.
+  // Otherwise, the clipping planes are passed down from a tileset.
+  const clippingPlanes = options.clippingPlanes;
+  if (defined(clippingPlanes) && clippingPlanes.owner === undefined) {
+    ClippingPlaneCollection.setOwner(clippingPlanes, this, "_clippingPlanes");
+  } else {
+    this._clippingPlanes = clippingPlanes;
+  }
```

Comment:
We decided that it would be better to handle this with some sort of reference count mechanism. I'll write up the discussion in a separate issue and link it here.

Reply:
Yesterday afternoon I opened https://github.com/CesiumGS/cesium/issues/10257 for this.